Content/default/webpack.config.js

```diff
@@ -24,21 +29,17 @@ var CONFIG = {
         // redirect requests that start with /api/ to the server on port 8085
```

Comment:
same as with minimal, dev server proxy doesn't seem to work with default template:

```
client: <e> [webpack-dev-server] [HPM] Error occurred while proxying request localhost:8080/api/ITodosApi/getTodos to http://localhost:8085/ [ECONNREFUSED] (https://nodejs.org/api/errors.html#errors_common_system_errors)
```

and before that client prints following to stderr:

```
client: <i> [webpack-dev-server] [HPM] Proxy created: /api/**  -> http://localhost:8085
client: <i> [webpack-dev-server] [HPM] Proxy created: /socket/**  -> http://localhost:8085
client: <i> [webpack-dev-server] Project is running at:
client: <i> [webpack-dev-server] Loopback: http://localhost:8080/
client: <i> [webpack-dev-server] On Your Network (IPv4): http://10.9.136.121:8080/
client: <i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:8080/
client: <i> [webpack-dev-server] Content not from webpack is served from '/Users/theimowski/oss/SAFE-template/Content/default/src/Client/public' directory
client: <i> [webpack-dev-server] 404s will fallback to '/index.html'
```

Reply:
That's odd, it seemed to work for me yesterday when I tried it out before pushing. I'll investigate, thanks.

Reply:
@theimowski do you have the latest LTS release of node installed? Just trying to work out why you are seeing these issues as I'm not. Are you running the projects from the Content folder directly, or packaging first or perhaps installing the template?

Reply:
I'm running node 17.0.1 and npm 8.1.0 on MacOS, and testing directly from Content directory

Reply:
Right, I have now downgraded to node 16.13.1, and all works fine. I think I upgraded node recently for some reason to v17, but as v16 is the LTS we could force it for SAFE template - see #489

Reply:
Ah excellent, glad we worked it out. Maybe it is worth a note somewhere (quickstart?) that warns people of this as I am sure many will hit it.

Reply:
Can confirm, it works with node `16.13.1` but not `17.2.0`. The client prints the same lines (`<i>`) to stderr in both cases.

Reply:
Looks like it's probably due to this https://github.com/nodejs/node/issues/40537

The problem is that when the dev server proxies to `http://localhost:8085`, in node 16 it would use ipv4 but in node 17 it defaults to ipv6. However, the server is listening only on ipv4 (`0.0.0.0`).

The fix is to make the server listen on all interfaces, including ipv6. So change `url` in `Server.fs` to `url "http://*:8085"`

Reply:
If one of you could try that and tell me if it works, I'll make the change and push it up. Thanks!

Reply:
see https://github.com/SAFE-Stack/SAFE-template/pull/490